lib/oneview-sdk/resource/api200/volume.rb

```diff
@@ -147,8 +147,9 @@ def get_snapshots
       # Gets all the attachable volumes managed by the appliance
       # @param [OneviewSDK::Client] client The client object for the OneView appliance
       # @return [Array<OneviewSDK::Volume>] Array of volumes
-      def self.get_attachable_volumes(client)
-        uri = "#{BASE_URI}/attachable-volumes"
+      def self.get_attachable_volumes(client, query = nil)
```

Comment:
Checked the API600 spec, query params: connections, networks were introduced in API600. So this method should be overwritten for 600 api.

Reply:
It is not specific to `connections`. Added here so that it can support other query parameters like `count`, `sort` etc as well.

Reply:
ok. got it.